openedx/features/survey_report/views.py

```diff
@@ -25,3 +26,16 @@ def post(self, _request):
         """
         generate_survey_report.delay()
         return redirect("admin:survey_report_surveyreport_changelist")
+
+
+class SurveyReportUpload(View):
+    """
+    View for send Survey Reports.
+    """
+    @method_decorator(login_required)
+    def get(self, _request, report_id):
```

Comment:
For security reasons, we should avoid having side-effects like these triggered by GET requests as much as possible. You can write this as a Django Admin [action](https://docs.djangoproject.com/en/3.2/ref/contrib/admin/actions/#writing-actions) instead.

Reply:
@ormsbee done.